### PR TITLE
Calculate_ch.1

### PR DESCRIPTION
**Pull request by Regggis**
Every functionality added

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @Regggis

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Regggis for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,44 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
+    int r;
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:5]: What is 'r'? Variable should always represent real operation. Like `result` `divider` `multiplier` `error_code` etc...

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+    std::string result;
+    if (command == "add")
+    {
+        r = first + second;
+        result =std::to_string(r);
+        return result;
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:11]: If you return variable here, you don't need to use `else if`

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
+    }
+    else if(command == "multiply")
+    {
+        r = first * second;
+        result =std::to_string(r);
+        return result;
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:17]: Same here

Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:17]: Just `return std::to_string(first * second);`

File: homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -2,6 +2,44 @@
...
+        {
+            result = "Division by 0";
+            return result;
+        }
+        else
+        {
+            r = first / second;
+            result =std::to_string(r);
+            return result;
+        }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:31]: else is redundant

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**